templates/http/default.conf.j2

```diff
@@ -12,8 +12,39 @@ upstream {{ item.value.upstreams[upstream].name }} {
     sticky cookie srv_id expires=1h  path=/;
 {% endif %}
 }
+{% endfor %}
+{% endif %}
 
+{% if (item.value.reverse_proxy_enable is defined) and item.value.reverse_proxy_enable %}
+{% if (item.value.reverse_proxy.proxy_cache_enable is defined) and item.value.reverse_proxy.proxy_cache_enable %}
```

Comment:
Same with the `proxy_cache_enable` variable.  You're already using other conditional checks so tweaking this shouldn't be too hard.

Reply:
Done